pkg/proxy/ipvs/proxier.go

```diff
@@ -578,30 +578,12 @@ func (handle *LinuxKernelHandler) GetModules() ([]string, error) {
 	}
 	ipvsModules := utilipvs.GetRequiredIPVSModules(kernelVersion)
 
-	builtinModsFilePath := fmt.Sprintf("/lib/modules/%s/modules.builtin", kernelVersionStr)
-	b, err := ioutil.ReadFile(builtinModsFilePath)
-	if err != nil {
-		klog.Warningf("Failed to read file %s with error %v. You can ignore this message when kube-proxy is running inside container without mounting /lib/modules", builtinModsFilePath, err)
-	}
 	var bmods []string
-	for _, module := range ipvsModules {
-		if match, _ := regexp.Match(module+".ko", b); match {
-			bmods = append(bmods, module)
-		}
-	}
 
-	// Try to load IPVS required kernel modules using modprobe first
-	for _, kmod := range ipvsModules {
-		err := handle.executor.Command("modprobe", "--", kmod).Run()
-		if err != nil {
-			klog.Warningf("Failed to load kernel module %v with modprobe. "+
-				"You can ignore this message when kube-proxy is running inside container without mounting /lib/modules", kmod)
-		}
-	}
-
-	// Find out loaded kernel modules
+	// Find out loaded kernel modules. If this is a full static kernel it will thrown an error
 	modulesFile, err := os.Open("/proc/modules")
 	if err != nil {
+		klog.Warningf("Failed to read file /proc/modules with error %v. Kube-proxy doesn't support fully statically compiled kernel yet", err)
```

Comment:
My knowledge in this area is lacking a bit. Wondering if "fully statically compiled kernel" is the right term here -- wondering if something like this is bit clearer to users:
```
Kube-proxy requires loadable modules enabled in the kernel
```

Reply:
Done :)